app/models/linked_account.rb

```diff
@@ -0,0 +1,35 @@
+# rubocop:disable Metrics/LineLength
+# == Schema Information
+#
+# Table name: linked_accounts
+#
+#  id                 :integer          not null, primary key
+#  encrypted_token    :string
+#  encrypted_token_iv :string
+#  private            :boolean          default(TRUE), not null
+#  share_from         :boolean          default(FALSE), not null
+#  share_to           :boolean          default(FALSE), not null
+#  sync_to            :boolean          default(FALSE), not null
+#  type               :string           not null
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#  external_user_id   :string           not null
+#  user_id            :integer          not null, indexed
+#
+# Indexes
+#
+#  index_linked_accounts_on_user_id  (user_id)
+#
+# Foreign Keys
+#
+#  fk_rails_166e103170  (user_id => users.id)
+#
+# rubocop:enable Metrics/LineLength
+
+class LinkedAccount < ApplicationRecord
+  belongs_to :user, required: true
+  # encyrpt the token
+  attr_encrypted :token, key: Base64.encode64(ENV['ATTR_ENCRYPT_KEY'])
```

Comment:
I _think_ what are we going to be storing in heroku (will it be an encoded or decoded key?)